panda/src/express/nodeReferenceCount.I

```diff
@@ -52,8 +52,6 @@ NodeReferenceCount(const NodeReferenceCount &copy) : ReferenceCount(copy) {
  */
 INLINE void NodeReferenceCount::
 operator = (const NodeReferenceCount &copy) {
-  nassertv(this != NULL);
```

Comment:
Resolution: remove (likely optimized out on many platforms; its presence may actually hinder, rather than help, investigation of mysterious segfaults)